.flake8

```diff
@@ -1,3 +1,12 @@
 [flake8]
-max-line-length = 120
-ignore = E501
+# The following are ignored:
```

Comment:
It's not obvious right away why are these settings in `[flake8]` section but the links in comments point to `black`. Perhaps add ` to work better with "black"` before the colon.